wallet-core/src/main/java/eu/europa/ec/eudi/wallet/transfer/openid4vp/OpenId4vpManager.kt

```diff
@@ -315,6 +356,63 @@ class OpenId4vpManager(
         }
     }
 
+    private fun mDocVPTokenConsensus(
+        deviceResponse: ByteArray,
+        resolvedRequestObject: ResolvedRequestObject.OpenId4VPAuthorization
+    ): Consensus.PositiveConsensus.VPTokenConsensus {
+        val vpToken =
+            Base64.getUrlEncoder().withoutPadding()
+                .encodeToString(deviceResponse)
+        logger?.d(TAG, "VpToken: $vpToken")
```

Comment:
Should this really be logged?

Reply:
Der logger ist aus der originalen lib selbst, den haben wir nicht selbst hinzugefügt